show/main.py

```diff
@@ -228,6 +228,45 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+# 'neighbor' subcommand ("show interfaces neighbor")
```

Comment:
>show interfaces neighbor [](start = 26, length = 24)

Suggest command line:
show interfaces neighbor expected.

In future, we may have other subcommand to observe the reality.
show interfaces neighbor arp
show interfaces neighbor lldp

Reply:
updated the cmd name as recommended